src/Fixer/Basic/BracesFixer.php

```diff
@@ -846,6 +846,11 @@ private function ensureWhitespaceAtIndexAndIndentMultilineComment(Tokens $tokens
 
         $nextToken = $tokens[$nextTokenIndex];
         if ($nextToken->isComment()) {
+            $previousToken = $tokens[$nextTokenIndex - 1];
+            // do not indent comments starting at the beginning of line - non-indented comments should not change indentation
+            if ($previousToken->isWhitespace() && 1 === preg_match('/\R$/', $previousToken->getContent())) {
+                return;
```

Comment:
I see no point of having that config.
Commented code is temporary code, either remove it before commiting or indent it properly.

I have understood #2986 bug that only first line of comment is aligned and not all of them, why you treat is as no alignment shall be done.
With later, you won't fix real comments in favor of not touching temporary commented code that shall not be part of the repo in first place.

If intention of this PR is to do the later (not touch comments without indentation), I'm big :-1: for it, due to reasoning above.